pkg/azuredisk/azuredisk_v2.go

```diff
@@ -278,16 +296,22 @@ func (d *DriverV2) StartControllersAndDieOnExit(ctx context.Context) {
 
 	// Setup a new controller to clean-up AzDriverNodes
 	// objects for the nodes which get deleted
-	klog.V(2).Info("Initializing AzDriverNode controller")
-	err = controller.InitializeAzDriverNodeController(mgr, &d.azDiskClient, d.objectNamespace)
+	// klog.V(2).Info("Initializing AzDriverNode controller")
+	// // err = controller.InitializeAzDriverNodeController(mgr, &d.azDiskClient, d.objectNamespace)
+	// // if err != nil {
+	// // 	klog.Errorf("Failed to initialize AzDriverNodeController. Error (%v). Exiting application...", err)
+	// // 	os.Exit(1)
+	// // }
+	klog.V(2).Info("Initializing AzVolumeAttachment controller")
+	err = controller.NewAzVolumeAttachmentController(ctx, mgr, &d.azDiskClient, d.objectNamespace)
 	if err != nil {
-		klog.Errorf("Failed to initialize AzDriverNodeController. Error (%v). Exiting application...", err)
+		klog.Errorf("Failed to initialize AzVolumeAttachment. Error (%v). Exiting application...", err)
 		os.Exit(1)
 	}
 
 	klog.V(2).Info("Starting controller manager")
 	if err := mgr.Start(ctx); err != nil {
-		klog.Errorf("Controller manager exited.")
+		klog.Errorf("Controller manager exited: (%v)", err)
```

Comment:
np: Parentheses are not necessary around the error.

Reply:
Done. I removed the parenthesis from the errors for other error comments too, but I will double check to see if I removed all of them.

Reply:
The parentheses still seem to be there.